### PR TITLE
Allow 2-5 fingers to swipe and pinch

### PR DESCRIPTION
**Pull request by ichaoX**
Relates to: #25

### CODE REVIEW

**Comment by JoseExposito** [comment]
Hi @ichaoX 

Thanks for the pull request. As I mentioned in #25, gestures performed with 5 fingers or 2 fingers in case of pinch, are only available for touchscreens. Including them would probably confuse some users.

I like you approach to clarify in the title that they are touchscreen only but, what do you think about putting the new options inside an expander?

https://docs.gtk.org/gtk4/class.Expander.html

I think that it could be easier for the users to understand that those are touchscreen only settings.

File: src/main-view/content/pinch-view.js [hunk 1]
```diff
@@ -65,6 +73,10 @@ class PinchView extends Gtk.ScrolledWindow {
 
     log('PinchView: Loading 4 fingers pinch');
     this.list4.showGestures(gestures4);
+
```
Comment (JoseExposito) [inline src/main-view/content/pinch-view.js:76]: Please remove the trailing white spaces.

File: src/main-view/content/pinch-view.js [hunk 2]
```diff
@@ -65,6 +73,10 @@ class PinchView extends Gtk.ScrolledWindow {
 
     log('PinchView: Loading 4 fingers pinch');
     this.list4.showGestures(gestures4);
+    
+    log('PinchView: Loading 5 fingers pinch');
+    this.list5.showGestures(gestures5);
+
```
Comment (JoseExposito) [inline src/main-view/content/pinch-view.js:79]: Extra line

**Review by JoseExposito — CHANGES_REQUESTED** [review]

**Comment by ichaoX** [comment]
Only available for touchscreens gestures have been put into expander.

**Comment by JoseExposito** [comment]
Sorry for the delay looking into this, it's been a busy week. Merged, thanks a lot!

**Pull request merged**